signers/nodes/mainnet/config.toml

```diff
@@ -0,0 +1,19 @@
+[Node.P2P]
+StaticNodes = [
+  "enode://ce8fa267164522976322a3ac91d6869fdec1acefbe065e9d60c11d82168b86f1a7feecd6fbfcb0dd6d82b93e9bdd063d7c0216419ee70236c8c13cd6ded4af73@159.89.188.156:30303",
```

Comment:
Should we dns these hosts rather than use IPs?

Reply:
IIRC it didn't work with the bootnode, so it might require code changes